From 4cdda601897999f41a911fee12d39a8efec3d0cc Mon Sep 17 00:00:00 2001
From: Sylvain Munaut <tnt@246tNt.com>
Date: Mon, 25 Aug 2014 21:12:07 +0200
Subject: [PATCH 1/2] python: Remove an old workaround for a swig bug

This has been removed of Gnuradio iteself in 2012 and this seems to
be causing issues for some OOT on some platforms ...

Signed-off-by: Sylvain Munaut <tnt@246tNt.com>
---
 python/__init__.py |   24 ------------------------
 1 file changed, 24 deletions(-)

diff --git a/python/__init__.py b/python/__init__.py
index 7f6f3ee..63a8e51 100644
--- a/python/__init__.py
+++ b/python/__init__.py
@@ -23,32 +23,8 @@ This is the GNU Radio IQBALANCE module. Place your Python package
 description here (python/__init__.py).
 '''
 
-# ----------------------------------------------------------------
-# Temporary workaround for ticket:181 (swig+python problem)
-import sys
-_RTLD_GLOBAL = 0
-try:
-    from dl import RTLD_GLOBAL as _RTLD_GLOBAL
-except ImportError:
-    try:
-	from DLFCN import RTLD_GLOBAL as _RTLD_GLOBAL
-    except ImportError:
-	pass
-
-if _RTLD_GLOBAL != 0:
-    _dlopenflags = sys.getdlopenflags()
-    sys.setdlopenflags(_dlopenflags|_RTLD_GLOBAL)
-# ----------------------------------------------------------------
-
-
 # import swig generated symbols into the iqbalance namespace
 from iqbalance_swig import *
 
 # import any pure python here
 #
-
-# ----------------------------------------------------------------
-# Tail of workaround
-if _RTLD_GLOBAL != 0:
-    sys.setdlopenflags(_dlopenflags)      # Restore original flags
-# ----------------------------------------------------------------
-- 
1.7.10.4

